pkg/client/password.go

```diff
@@ -6,12 +6,26 @@ import (
 	"github.com/howeyc/gopass"
 )
 
+const (
+	minPassLen = 8
+)
+
+func checkPassword(pass string) error {
+	if len([]rune(pass)) < minPassLen {
+		return fmt.Errorf("minimum length is %d", minPassLen)
+	}
+	return nil
+}
```

Comment:
@drgarcia1986 updated. I guess creating specific errors in this case is not worthwhile so I only checked for error and success.